src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -6612,14 +6687,9 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
             tmp = nullptr;
             return ;
         }
-        // Keyword arguments handled in make_call_helper
-        #define parse_args() if( x.n_keywords == 0 ) { \
-            args.reserve(al, x.n_args); \
-            visit_expr_list(x.m_args, x.n_args, args); \
-        } \
```

Comment:
Here, I was overlooking the first line `if( x.n_keywords == 0 ) { \`. Hence, I refactored this into a function, which I am hoping is more readable.